README.md

```diff
@@ -247,6 +248,7 @@ PUBLISH OPTIONS:
         --publish-as-is          Publish crates from the current commit without versioning
         --registry <REGISTRY>    The Cargo registry to use for publishing
         --token <TOKEN>          The token to use for publishing
+        --dry-run                Perform checks without uploading. This option is WIP and performs less checks than `cargo publish --dry-run`.
```

Comment:
```suggestion
        --dry-run                Perform checks without uploading. WIP and performs fewer checks than `cargo publish --dry-run`.
```

Reply:
Sorry, wrong place to comment. This needs to be fixed in the comment for this option definition.